iml-manager-cli/src/main.rs

```diff
@@ -86,9 +86,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     match matches {
         App::Shell { .. } => (),
-        _ => {
-            dotenv::from_path("/var/lib/chroma/iml-settings.conf").expect("Could not load cli env")
-        }
+        _ => dotenv::from_path("/etc/emf/emf-settings.conf")
+            .or_else(|_| dotenv::from_path("/etc/iml/iml-settings.conf"))
+            .or_else(|_| dotenv::from_path("/var/lib/chroma/iml-settings.conf"))
```

Comment:
I understand emf/iml difference, but why the third variant?

Shouldn't the name of the configuration file depend on selfname?
Shouldn't be the files read in order, overriding any previous options, at least the one under /var?

Reply:
Third is for docker setup.

I don't think we should override anything.